packages/calcite-components/src/components/rating/rating.scss

```diff
@@ -4,6 +4,11 @@
  * These properties can be overridden using the component's tag as selector.
  *
  * @prop --calcite-rating-spacing-unit: The amount of left and right margin spacing between each rating star.
+ * @prop --calcite-rating-hover-selected-color: Specifies the component's color when hovered and selected.
```

Comment:
I think the order of this should be `--calcite-rating-selected-color-hover` right?

Reply:
I updated this to `--calcite-rating-selected-color-hover`